subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/HttpElasticClient.kt

```diff
@@ -36,26 +36,32 @@ internal class HttpElasticClient(
     private val timestampFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSSZ")
 
     override fun sendMessage(
-        tag: String,
         level: String,
         message: String,
+        metadata: Map<String, String>,
         throwable: Throwable?
     ) {
         try {
             val nowWithTimeZone = ZonedDateTime.ofInstant(timeProvider.now().toInstant(), timezone)
             val nowWithoutTimeZone = LocalDateTime.ofInstant(timeProvider.now().toInstant(), timezone)
 
+            val params = mutableMapOf(
+                "@timestamp" to nowWithTimeZone.format(timestampFormatter),
+                "level" to level,
+                "build_id" to buildId,
+                "message" to message
+            )
+
+            if (!throwable?.message.isNullOrBlank()) {
+                params["error_message"] = throwable?.message
```

Comment:
What about full stacktrace with cases?
Should we look for them only in Sentry?

Reply:
i think yes, it's more convenient there; but we don't have a way to go from kibana to sentry directly
we don't have access to warning level stacktraces for now; is it ok? or send them to sentry also?

Reply:
I think If we are able to connect kibana event with a sentry event we should send stacktraces only to the sentry

Reply:
it's anyway too big and unreadable for kibana

for now user scenario is:
1. user goes to kibana analyzing logs
1. if stacktrace needed user should search sentry for same error, narrowing search with build_id, tag etc